lib/clerk/rack_middleware_v2.rb

```diff
@@ -163,11 +163,14 @@ def call(env)
       end
 
       token = verify_token(cookie_token)
+      return signed_out(env) unless token
```

Comment:
IIRC, `verify_token() == false` can also happen if the token is expired, for example. So, if I'm signed in and suddenly I close my browser, and come back after 1 hour, then `verify_token()` will return `false` here. In that case, don't we want to show the interstitial so that the token is refreshed (the long-lived FAPI cookie will still be there) and I remain signed in?

Reply:
In case we jwt is expired we re-raise the `ExpiredSignature` error instead of returning `false` from `verify_token` . see `rescue JWT::ExpiredSignature, JWT::InvalidIatError => e` below.